proto/api/v2/compliance_results_service.proto

```diff
@@ -51,11 +52,15 @@ message ComplianceScanResult {
 
 // ComplianceScanStatsShim models statistics of checks for a given scan configuration
 message ComplianceScanStatsShim {
-    string                                  scan_name                = 1;
-    int32                                   number_of_checks         = 2;
-    int32                                   number_of_failing_checks = 3;
-    int32                                   number_of_passing_checks = 4;
-    google.protobuf.Timestamp               last_scan                = 5;
+    string                                  scan_name = 1;
+
+    // Group the number of occurrences by status
+    message ComplianceCheckStatusCount {
```

Comment:
we have more states than just pass and fail.  So returning counts of each state and the caller whether UI or API user can determine what they mean for their purposes.